sympy/functions/elementary/trigonometric.py

```diff
@@ -2867,7 +2867,7 @@ class atan2(InverseTrigonometricFunction):
 
     >>> from sympy import atan
     >>> atan2(y, x).rewrite(atan)
-    2*atan(y/(x + sqrt(x**2 + y**2)))
+    Piecewise((2*atan(y/(x + sqrt(x**2 + y**2))), Ne(y, 0)), (pi, x < 0), (0, x > 0), (nan, True))
```

Comment:
I'm surprised that there isn't a doctest failure for Ne and nan not being imported here. Am I missing something or are they not imported in this docstring?

Reply:
Oh of course they're in the output from the doctest!